src/components/Layout/MainNavbar/ConnectionIndicator.tsx

```diff
@@ -1,33 +1,136 @@
-import React from "react";
+import React, { useState, useEffect, useRef } from "react";
 // @ts-ignore
 import { NavItem, NavLink, Badge } from "shards-react";
+import { Popover, Overlay } from "react-bootstrap";
+import { Row, Col, Button, Form } from "shards-react";
+import { Store, Dispatcher, Constants } from "../../../flux";
+import BaseFormTextInput from "components/Common/BaseFormTextInput";
```

Comment:
Could you change the name of the default export in the file BaseFormTextInput to the file name?

Reply:
Yes sure. Done. Thank you